modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -480,6 +495,101 @@ inline IE::Blob::Ptr extractBlob(IECallContext& ctx, std::size_t i) {
 }
 } // anonymous namespace
 
+class cv::gimpl::ie::RequestPool {
+public:
+    using CompiledPtr = std::shared_ptr<cv::gimpl::ie::IECompiled>;
+    using Run      = std::function<void(cv::gimpl::ie::IECompiled&)>;
+    using Callback = std::function<void(cv::gimpl::ie::IECompiled&)>;
+    struct Task {
+        Run run;
+        Callback callback;
+    };
```

Comment:
So what is `run`, what is `callback`, what is the idea behind this? A comment is required.

Reply:
Added comment